client/server/pages/index.js

```diff
@@ -290,8 +290,9 @@ function setUpLoggedOutRoute( req, res, next ) {
 		setupRequests.push( setUpLocalLanguageRevisions( req ) );
 	}
 
-	if ( req.cookies?.subkey && ! req.context.user ) {
+	if ( req.cookies?.subkey ) {
```

Comment:
@TimBroddin, I am removing this check in case anyone else wants to store something else inside the user object for non-logged in user. It should not break the subscription management portal. In case you were worried this code may get executed for logged-in users, it does not. 🙂